package-lock.json

```diff
@@ -20862,4 +20862,4 @@
       }
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
Minor thing, but can you get changes in package-lock removed from the commit?

You can reset the HEAD, remove from staging and do a force push.